samples/otlptracehttp/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "sample-app",
+  "version": "0.1.0",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "main": "index.js",
+  "scripts": {
+    "start": "node ./app.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@opentelemetry/api": "^1.8.0",
+    "@opentelemetry/exporter-metrics-otlp-http": "^0.49.1",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.49.1",
+    "@opentelemetry/resources": "^1.22.0",
+    "@opentelemetry/sdk-node": "^0.49.1",
+    "@opentelemetry/semantic-conventions": "^1.22.0",
+    "express": "^4.18.3",
+    "google-auth-library": "^9.6.3"
```

Comment:
Since this is an example, can you remove the carats so that it installs exact versions?

One other thing, this repo uses npm workspaces where there is a single package-lock.json in the repo root and all of the packages get linked together (similar to go workspaces). Can you delete the package-lock.json and `node_modules` from this directory and instead run `npm install` from the repo root?

Reply:
Done, please check if I missed anything